### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#63679]
#### VFS team name
Debt Resolution

#### VFS product name
Financial Status Report (FSR)

#### VFS feature name
Streamlined Waiver

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
In Step 2, after entering your work history when reviewing the entries as cards, the headings aren't nested properly. "Your work history" is an H3, so each card describing a work history entry should be nested beneath that. Instead, the heading for each card is an H3.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
Recommend the headings for each card be changed to H4s. This matches with similar occurences elsewhere in the product, so this looks like it was probably just a typo.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by HeatherRienks** [comment]
This work was completed in #64423. Closing ticket.

cc: @Tom-Davis
